docs/pipelines/evaluation.md

```diff
@@ -115,12 +115,12 @@ Let's look at gsm8k to understand a bit more how each part of the evaluation wor
 Inside [nemo_skills/dataset/gsm8k/\_\_init\_\_.py](https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/dataset/gsm8k/__init__.py) we see the following
 
 ```python
-from nemo_skills.evaluation.metrics import MathMetrics
+from nemo_skills.evaluation.metrics import get_metrics
 
 # settings that define how evaluation should be done by default (all can be changed from cmdline)
 PROMPT_CONFIG = 'generic/math'
 DATASET_GROUP = 'math'
-METRICS_CLASS = MathMetrics
+METRICS_CLASS = get_metrics("metric-math")
```

Comment:
Let's maybe not call it here, so that metrics creation can be done inside compute metrics. So here you can just have it as 
```suggestion
METRICS = "metric-math"
```
and then you would call get_metrics(METRICS) later when you actually need to use that class

Reply:
Done

Reply:
Done